.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Report an issue
+---
+
+# Bug Report #
+
+## Environment ##
+
+What operating system is being assembled?
```

Comment:
From various IR/slack chats, I think there are some additional details that often make the key difference:
 * host OS
 * docker or podman
 * if podman, privileged or unprivileged